validphys2/src/validphys/checks.py

```diff
@@ -44,8 +44,7 @@ def check_pdf_is_montecarlo(ns, **kwargs):
 
 
 @make_argcheck
-def check_pdf_is_montecarlo_or_symmhessian(ns, **kwargs):
-    pdf = ns['pdf']
+def check_pdf_is_montecarlo_or_symmhessian(pdf, **kwargs):
     etype = pdf.error_type
     check(
         etype in {'replicas', 'symhessian'},
```

Comment:
Hi @scarlehoff, spotted another bug (my bad)
```suggestion
        etype in {'replicas', 'symmhessian'},
```